docs/user/commandline_old_commands.rst

```diff
@@ -0,0 +1,72 @@
+Old Command Line Commands
+=========================
+
+In GeoMesa 1.3.0 some command names have changed. Please reference the chart below for more information.
+
+Accumulo Command Line Tools
+---------------------------
+
++---------------------+--------------------+
+| Old                 | New                |
++=====================+====================+
+| add-attribute-index	| add-attributeindex |
++---------------------+--------------------+
+| add-index         	| add-index          |
```

Comment:
Did this change?  If not, do we need it in the table (or is there a better way to indicate that it remained the same)?